TSPL.docc/LanguageGuide/TheBasics.md

```diff
@@ -2,25 +2,23 @@
 
 Work with common kinds of data and write basic syntax.
 
-Swift is a programming language for iOS, macOS, watchOS, and tvOS app development.
-If you have experience developing in C or Objective-C,
-many parts of Swift will be familiar to you.
+Swift is a programming language designed for developing many types of software, from mobile and desktop apps, to server, system, and embedded software.
+Many parts of Swift will be familiar if you have experience developing in other popular languages.
 
-Swift provides its own versions of all fundamental C and Objective-C types,
+Swift provides its own versions of fundamental types,
 including `Int` for integers, `Double` and `Float` for floating-point values,
 `Bool` for Boolean values, and `String` for textual data.
 Swift also provides powerful versions of the three primary collection types,
 `Array`, `Set`, and `Dictionary`,
 as described in <doc:CollectionTypes>.
 
-Like C, Swift uses variables to store and refer to values by an identifying name.
+Swift uses variables to store and refer to values by an identifying name.
 Swift also makes extensive use of variables whose values can't be changed.
-These are known as constants, and are much more powerful than constants in C.
-Constants are used throughout Swift to make code safer and clearer in intent
+These are known as constants, and are used throughout Swift to make code safer and clearer in intent
```

Comment:
I'd prefer not to mark "constants" as a new term here, even though it is being introduced and defined, because there's a whole section about it immediately after the intro.

Reply:
OK